{"name": "Проверка перехода к разделу  \"История заказов\" под пользователями: без заказов и с заказами", "status": "passed", "description": "Открываем сайт, переходим на страницу входа и логинимся. нажимаем кнопку \"Личный кабинет\" и переходим на вкладку \"История заказов\". Проверяем наличие контейнера заказов", "steps": [{"name": "Переход в раздел \"История заказов\"", "status": "passed", "steps": [{"name": "Переход по кнопке на другую страницу", "status": "passed", "steps": [{"name": "Клик по элементу с обработкой исключения", "status": "passed", "steps": [{"name": "поиск элемента с нажатием на него", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementClickInterceptedException: Message: Element <a class=\"AppHeader_header__link__3D_hX\" href=\"/account\"> is not clickable at point (1176,44) because another element <div class=\"Modal_modal_overlay__x2ZCr\"> obscures it\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nElementClickInterceptedError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:337:5\nwebdriverClickElement@chrome://remote/content/marionette/interaction.sys.mjs:177:11\ninteraction.clickElement@chrome://remote/content/marionette/interaction.sys.mjs:136:11\nclickElement@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:205:29\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:85:31\n\n", "trace": "  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\pages\\base_page.py\", line 30, in click_to_element\n    self.find_element_with_wait(locator).click()\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 94, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 395, in _execute\n    return self._parent.execute(command, params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Поиск элемента с ожиданием", "status": "passed", "parameters": [{"name": "locator", "value": "('xpath', './/a[@href=\"/account\"]')"}], "start": 1726550216037, "stop": 1726550216053}], "parameters": [{"name": "locator", "value": "('xpath', './/a[@href=\"/account\"]')"}], "start": 1726550216037, "stop": 1726550216083}, {"name": "поиск элемента с нажатием на него", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementClickInterceptedException: Message: Element <a class=\"AppHeader_header__link__3D_hX\" href=\"/account\"> is not clickable at point (1176,44) because another element <div class=\"Modal_modal_overlay__x2ZCr\"> obscures it\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nElementClickInterceptedError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:337:5\nwebdriverClickElement@chrome://remote/content/marionette/interaction.sys.mjs:177:11\ninteraction.clickElement@chrome://remote/content/marionette/interaction.sys.mjs:136:11\nclickElement@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:205:29\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:85:31\n\n", "trace": "  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\pages\\base_page.py\", line 30, in click_to_element\n    self.find_element_with_wait(locator).click()\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 94, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 395, in _execute\n    return self._parent.execute(command, params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Поиск элемента с ожиданием", "status": "passed", "parameters": [{"name": "locator", "value": "('xpath', './/a[@href=\"/account\"]')"}], "start": 1726550216092, "stop": 1726550216109}], "parameters": [{"name": "locator", "value": "('xpath', './/a[@href=\"/account\"]')"}], "start": 1726550216092, "stop": 1726550216116}, {"name": "поиск элемента с нажатием на него", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementClickInterceptedException: Message: Element <a class=\"AppHeader_header__link__3D_hX\" href=\"/account\"> is not clickable at point (1176,44) because another element <div class=\"Modal_modal_overlay__x2ZCr\"> obscures it\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nElementClickInterceptedError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:337:5\nwebdriverClickElement@chrome://remote/content/marionette/interaction.sys.mjs:177:11\ninteraction.clickElement@chrome://remote/content/marionette/interaction.sys.mjs:136:11\nclickElement@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:205:29\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:85:31\n\n", "trace": "  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\pages\\base_page.py\", line 30, in click_to_element\n    self.find_element_with_wait(locator).click()\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 94, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 395, in _execute\n    return self._parent.execute(command, params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Поиск элемента с ожиданием", "status": "passed", "parameters": [{"name": "locator", "value": "('xpath', './/a[@href=\"/account\"]')"}], "start": 1726550216129, "stop": 1726550216146}], "parameters": [{"name": "locator", "value": "('xpath', './/a[@href=\"/account\"]')"}], "start": 1726550216129, "stop": 1726550216153}, {"name": "поиск элемента с нажатием на него", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementClickInterceptedException: Message: Element <a class=\"AppHeader_header__link__3D_hX\" href=\"/account\"> is not clickable at point (1176,44) because another element <div class=\"Modal_modal_overlay__x2ZCr\"> obscures it\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nElementClickInterceptedError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:337:5\nwebdriverClickElement@chrome://remote/content/marionette/interaction.sys.mjs:177:11\ninteraction.clickElement@chrome://remote/content/marionette/interaction.sys.mjs:136:11\nclickElement@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:205:29\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:85:31\n\n", "trace": "  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\pages\\base_page.py\", line 30, in click_to_element\n    self.find_element_with_wait(locator).click()\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 94, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 395, in _execute\n    return self._parent.execute(command, params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Поиск элемента с ожиданием", "status": "passed", "parameters": [{"name": "locator", "value": "('xpath', './/a[@href=\"/account\"]')"}], "start": 1726550216167, "stop": 1726550216190}], "parameters": [{"name": "locator", "value": "('xpath', './/a[@href=\"/account\"]')"}], "start": 1726550216167, "stop": 1726550216197}, {"name": "поиск элемента с нажатием на него", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementClickInterceptedException: Message: Element <a class=\"AppHeader_header__link__3D_hX\" href=\"/account\"> is not clickable at point (1176,44) because another element <div class=\"Modal_modal_overlay__x2ZCr\"> obscures it\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nElementClickInterceptedError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:337:5\nwebdriverClickElement@chrome://remote/content/marionette/interaction.sys.mjs:177:11\ninteraction.clickElement@chrome://remote/content/marionette/interaction.sys.mjs:136:11\nclickElement@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:205:29\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:85:31\n\n", "trace": "  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\pages\\base_page.py\", line 30, in click_to_element\n    self.find_element_with_wait(locator).click()\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 94, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 395, in _execute\n    return self._parent.execute(command, params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Поиск элемента с ожиданием", "status": "passed", "parameters": [{"name": "locator", "value": "('xpath', './/a[@href=\"/account\"]')"}], "start": 1726550216209, "stop": 1726550216225}], "parameters": [{"name": "locator", "value": "('xpath', './/a[@href=\"/account\"]')"}], "start": 1726550216209, "stop": 1726550216232}, {"name": "поиск элемента с нажатием на него", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementClickInterceptedException: Message: Element <a class=\"AppHeader_header__link__3D_hX\" href=\"/account\"> is not clickable at point (1176,44) because another element <div class=\"Modal_modal_overlay__x2ZCr\"> obscures it\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nElementClickInterceptedError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:337:5\nwebdriverClickElement@chrome://remote/content/marionette/interaction.sys.mjs:177:11\ninteraction.clickElement@chrome://remote/content/marionette/interaction.sys.mjs:136:11\nclickElement@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:205:29\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:85:31\n\n", "trace": "  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\pages\\base_page.py\", line 30, in click_to_element\n    self.find_element_with_wait(locator).click()\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 94, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 395, in _execute\n    return self._parent.execute(command, params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Поиск элемента с ожиданием", "status": "passed", "parameters": [{"name": "locator", "value": "('xpath', './/a[@href=\"/account\"]')"}], "start": 1726550216246, "stop": 1726550216263}], "parameters": [{"name": "locator", "value": "('xpath', './/a[@href=\"/account\"]')"}], "start": 1726550216246, "stop": 1726550216269}, {"name": "поиск элемента с нажатием на него", "status": "passed", "steps": [{"name": "Поиск элемента с ожиданием", "status": "passed", "parameters": [{"name": "locator", "value": "('xpath', './/a[@href=\"/account\"]')"}], "start": 1726550216282, "stop": 1726550216295}], "parameters": [{"name": "locator", "value": "('xpath', './/a[@href=\"/account\"]')"}], "start": 1726550216282, "stop": 1726550216510}], "parameters": [{"name": "button", "value": "('xpath', './/a[@href=\"/account\"]')"}, {"name": "locator_exception", "value": "('xpath', './/*[@class=\"Modal_modal_overlay__x2ZCr\"]')"}], "start": 1726550216037, "stop": 1726550216510}, {"name": "Поиск элемента с ожиданием", "status": "passed", "parameters": [{"name": "locator", "value": "('xpath', './/li[contains(@class, \"Account_listItem\")]//a[text() = \"Профиль\"]')"}], "start": 1726550216510, "stop": 1726550216525}], "parameters": [{"name": "button", "value": "('xpath', './/a[@href=\"/account\"]')"}, {"name": "element", "value": "('xpath', './/li[contains(@class, \"Account_listItem\")]//a[text() = \"Профиль\"]')"}, {"name": "locator_exception", "value": "('xpath', './/*[@class=\"Modal_modal_overlay__x2ZCr\"]')"}], "start": 1726550216037, "stop": 1726550216525}, {"name": "Переход по кнопке на другую страницу", "status": "passed", "steps": [{"name": "Клик по элементу с обработкой исключения", "status": "passed", "steps": [{"name": "поиск элемента с нажатием на него", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementClickInterceptedException: Message: Element <a class=\"Account_link__2ETsJ text text_type_main-medium text_color_inactive\" href=\"/account/order-history\"> is not clickable at point (128,307) because another element <div class=\"Modal_modal_overlay__x2ZCr\"> obscures it\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nElementClickInterceptedError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:337:5\nwebdriverClickElement@chrome://remote/content/marionette/interaction.sys.mjs:177:11\ninteraction.clickElement@chrome://remote/content/marionette/interaction.sys.mjs:136:11\nclickElement@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:205:29\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:85:31\n\n", "trace": "  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\pages\\base_page.py\", line 30, in click_to_element\n    self.find_element_with_wait(locator).click()\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 94, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 395, in _execute\n    return self._parent.execute(command, params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Поиск элемента с ожиданием", "status": "passed", "parameters": [{"name": "locator", "value": "('xpath', './/a[text()=\"История заказов\"]')"}], "start": 1726550216525, "stop": 1726550216543}], "parameters": [{"name": "locator", "value": "('xpath', './/a[text()=\"История заказов\"]')"}], "start": 1726550216525, "stop": 1726550216550}, {"name": "поиск элемента с нажатием на него", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementClickInterceptedException: Message: Element <a class=\"Account_link__2ETsJ text text_type_main-medium text_color_inactive\" href=\"/account/order-history\"> is not clickable at point (128,307) because another element <div class=\"Modal_modal_overlay__x2ZCr\"> obscures it\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nElementClickInterceptedError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:337:5\nwebdriverClickElement@chrome://remote/content/marionette/interaction.sys.mjs:177:11\ninteraction.clickElement@chrome://remote/content/marionette/interaction.sys.mjs:136:11\nclickElement@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:205:29\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:85:31\n\n", "trace": "  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\pages\\base_page.py\", line 30, in click_to_element\n    self.find_element_with_wait(locator).click()\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 94, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 395, in _execute\n    return self._parent.execute(command, params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Поиск элемента с ожиданием", "status": "passed", "parameters": [{"name": "locator", "value": "('xpath', './/a[text()=\"История заказов\"]')"}], "start": 1726550216563, "stop": 1726550216585}], "parameters": [{"name": "locator", "value": "('xpath', './/a[text()=\"История заказов\"]')"}], "start": 1726550216563, "stop": 1726550216590}, {"name": "поиск элемента с нажатием на него", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementClickInterceptedException: Message: Element <a class=\"Account_link__2ETsJ text text_type_main-medium text_color_inactive\" href=\"/account/order-history\"> is not clickable at point (128,307) because another element <div class=\"Modal_modal_overlay__x2ZCr\"> obscures it\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nElementClickInterceptedError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:337:5\nwebdriverClickElement@chrome://remote/content/marionette/interaction.sys.mjs:177:11\ninteraction.clickElement@chrome://remote/content/marionette/interaction.sys.mjs:136:11\nclickElement@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:205:29\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:85:31\n\n", "trace": "  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\pages\\base_page.py\", line 30, in click_to_element\n    self.find_element_with_wait(locator).click()\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 94, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 395, in _execute\n    return self._parent.execute(command, params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Поиск элемента с ожиданием", "status": "passed", "parameters": [{"name": "locator", "value": "('xpath', './/a[text()=\"История заказов\"]')"}], "start": 1726550216604, "stop": 1726550216618}], "parameters": [{"name": "locator", "value": "('xpath', './/a[text()=\"История заказов\"]')"}], "start": 1726550216604, "stop": 1726550216622}, {"name": "поиск элемента с нажатием на него", "status": "passed", "steps": [{"name": "Поиск элемента с ожиданием", "status": "passed", "parameters": [{"name": "locator", "value": "('xpath', './/a[text()=\"История заказов\"]')"}], "start": 1726550216631, "stop": 1726550216645}], "parameters": [{"name": "locator", "value": "('xpath', './/a[text()=\"История заказов\"]')"}], "start": 1726550216631, "stop": 1726550216858}], "parameters": [{"name": "button", "value": "('xpath', './/a[text()=\"История заказов\"]')"}, {"name": "locator_exception", "value": "('xpath', './/*[@class=\"Modal_modal_overlay__x2ZCr\"]')"}], "start": 1726550216525, "stop": 1726550216858}, {"name": "Поиск элемента с ожиданием", "status": "passed", "parameters": [{"name": "locator", "value": "('xpath', './/div[contains(@class, \"Account_contentBox\")]')"}], "start": 1726550216858, "stop": 1726550216871}], "parameters": [{"name": "button", "value": "('xpath', './/a[text()=\"История заказов\"]')"}, {"name": "element", "value": "('xpath', './/div[contains(@class, \"Account_contentBox\")]')"}, {"name": "locator_exception", "value": "('xpath', './/*[@class=\"Modal_modal_overlay__x2ZCr\"]')"}], "start": 1726550216525, "stop": 1726550216871}], "start": 1726550216037, "stop": 1726550216871}], "attachments": [{"name": "log", "source": "f4625900-1049-45ec-9992-bcd15b3e830c-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "driver", "value": "'firefox'"}, {"name": "customer", "value": "'response_customer'"}], "start": 1726550215087, "stop": 1726550216871, "uuid": "0f662506-2c23-455c-a47a-f070cc975f37", "historyId": "25de67c2bd024c179a78e2e62ebd6486", "testCaseId": "106f9cff248f86c186ec3db8a890e3eb", "fullName": "tests.test_account_page.TestAccountPage#test_transition_order_history", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_account_page"}, {"name": "subSuite", "value": "TestAccountPage"}, {"name": "host", "value": "DESKTOP-Q7I2UDM"}, {"name": "thread", "value": "21020-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_account_page"}]}